src/themes/proficiency.css

```diff
@@ -179,6 +179,28 @@ body[data-theme="proficiency"] {
       font-weight: 700;
     }
   }
+
+  .sidebar-action-button {
+    width: 100%;
+    height: 36px;
+  }
```

Comment:
Tweak the sidebar action buttons per feedback.

![CleanShot 2568-01-14 at 22 01 41@2x](https://github.com/user-attachments/assets/0fa3196f-bada-4b55-bbe6-512d37738150)